packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/converter/convertHandles.ts

```diff
@@ -23,35 +23,43 @@ export const convertHandles = (gqlNode: GQLNode<GQLNodeStyle>, gqlEdges: GQLEdge
     if (edge.sourceId === gqlNode.id) {
       connectionHandles.push({
         id: `handle--source--${gqlNode.id}--${sourceHandlesCounter}`,
+        index: connectionHandles.filter((connectionHandle) => connectionHandle.position === Position.Right).length,
```

Comment:
sourceHandlesCounter represents the number of source handles, this represents the number of handles at the right side. The value actually does not matter much since we calculate the actual value later, so I  should probably just put 0

Reply:
OK.